### ISSUE TITLE
bug(mac): add timeout for accessibility check

### ISSUE THREAD

**Issue by sgschantz** [keymanapp/keyman#16579]
### Describe the bug

The Keyman Configuration app sends a notification to the Keyman input method to check the whether the user has granted Accessibility privileges for Keyman. It discovers the result via a notification sent in response, and in some cases it animates a spinner as it awaits the response. There should be some timeout enforced in case the response is not received.

### Keyman apps

- [ ] Keyman for Android
- [ ] Keyman for iPhone and iPad
- [ ] Keyman for Linux
- [x] Keyman for macOS
- [ ] Keyman for Windows
- [ ] Keyman Developer
- [ ] KeymanWeb
- [ ] Other - give details at bottom of form

### Keyman version

19 alpha
